### ISSUE TITLE
SG can't be installed as a service on Couchbase SG AMI

### ISSUE THREAD

**Issue by krugster** [couchbase/sync_gateway#1153]
There is an issue with the service install script on our own AMI that prevents SG from being installed as a service because it doesn't recognize the OS type of "Linux"

**Comment by zgramana** [comment]
The issue is that [we aren't handling the distribution check for custom (re)distributions of CentOS](https://github.com/couchbase/sync_gateway/blob/master/service/sync_gateway_service_install.sh#L37-L49), like Amazon Linux. For example, [for Amazon Linux we need do the following](http://superuser.com/questions/344837/determine-linux-distribution), at a minimum:

``` sh
    elif [ -f /etc/system-release ]; then
        OS=RedHat
        VER=`cat /etc/redhat-release | sed s/.*release\ // | sed s/\ .*//`
```

**Comment by ajres** [comment]
/etc/redhat-release is missing from the Amazon Linux distro, there is no command available to determine the underlying REdHat/CentOS version.

Amazon state that they try to achieve binary compatibility with at least RedHat 5 and more recently with RedHat 6.

Fix defaults to RedHat 5.0 for maximum compatibility

**Comment by ajres** [comment]
This has now been tested against an Amazon Linux distro and the Sync Gateway service can be successfully installed and started via sysv_init scripts

**Comment by ajres** [comment]
Merged to a new branch off master PR #1249 raised, closing this ticket out.
